src/demo/data.js

```diff
@@ -26,27 +30,112 @@ const enrollments = [...Array(numEnrollments)].map((_, idx) => {
     course_price: course.price,
     course_start: courseStart,
     course_end: courseEnd,
-    passed_timestamp: Math.random() <= hasPassed ? faker.date.between(courseStart, courseEnd)
+    passed_timestamp: hasPassed ? faker.date.recent(7)
       : null,
     has_passed: hasPassed,
     current_grade: grade / 100,
-    last_activity_date: faker.date.recent(Math.random() <= percentActive ? 7 : 45),
+    last_activity_date: lastActivityDate,
   };
 });
 
+const filterEnrolledLearners = () => {
+  return enrollments.map((enrollment) => {
+    return {
+      user_email: enrollment.user_email,
+      user_account_creation_timestamp: faker.date.past(),
+      enrollment_count: faker.random.number(1, 10), // TODO: another way to populate data?
```

Comment:
Should we have the enrollment count range include `0` to represent when some users don't have any enrollments?:

```javascript
faker.random.number(0, 10),
```